packages/components/src/sites-table/test/use-sites-table-filtering.tsx

```diff
@@ -16,35 +16,28 @@ function createMockSite( {
 	URL?: string;
 	is_private?: boolean;
 	is_coming_soon?: boolean;
-} = {} ): Parameters< typeof useSitesTableFiltering >[ 0 ][ number ] {
+} = {} ) {
 	return {
-		ID,
 		name: name ?? `site ${ ID }`,
 		URL: URL ?? `https://site${ ID }.io`,
 		slug: `site${ ID }.io`,
 		is_private,
 		is_coming_soon,
-		plan: {
-			product_id: 1,
-			product_slug: 'product_slug',
-			product_name_short: 'product_name_short',
-			expired: false,
-			user_is_owner: true,
-			is_free: true,
-		},
```

Comment:
@zaguiini Why'd you remove this data?

Reply:
Simply because we're not using this data in the component, and TS doesn't care about it.

Reply:
👍